test/integration/scripts/run-tests.sh

```diff
@@ -23,16 +23,32 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
+# setup hosts
+addHost() {
+  local hostname="$1"
+  
+  # check first to avoid sudo prompt if host is already set
+  if hostile list | grep -q "127.0.0.1 $hostname"; then
+    echo "$hostname already set"
+  else
+    sudo hostile set 127.0.0.1 "$hostname"
```

Comment:
```suggestion
  if pnpm hostile list | grep -q "127.0.0.1 $hostname"; then
    echo "$hostname already set"
  else
    sudo pnpm hostile set 127.0.0.1 "$hostname"
```
right? or did I not install it correctly?